data/coalition/lunarium intro.txt

```diff
@@ -1735,7 +1735,7 @@ mission "Lunarium: Combat Training 2"
 		event "lunarium training: shaula back to normal"
 		event "lunarium training done" 96 148
 		conversation
-			`Pyakri contacts you right as you land, by a text message in your monitor. "A great help you were, Captain! More... veteran members, if you could call them that, the crew I brought are, so much trouble with their morale, I had not, but still worried, the pirates had me. Used to human ships, we are not, so fly them well enough to combat such fleets, we cannot. Not yet. Now that guided us through this first fight, you have, work on our own for a while - taking smaller, simpler bounties - we will. Contact you again, I shall, once done with our training, we are."`
+			`Pyakri contacts you right as you land, by a text message in your monitor. "A great help you were, Captain! More... inveterate members, if call them that you could, the crew with me are, so have much trouble with their morale, I did not, but still have me worried, the pirates did. Used to human ships, we are not, so fly them well enough to combat such fleets, we cannot. Not yet. Now that guided us through this first fight, you have, work on our own for a while - taking smaller, simpler bounties - we will. Contact you again, I shall, once done with our training, we are."`
```

Comment:
Why the change from veteran to inveterate? Idk if it's just me but I had to google to find out what inveterate means, and even then I don't really see it as having the same intended meaning.